pkgs/applications/editors/jetbrains/default.nix

```diff
@@ -201,7 +199,8 @@ rec {
             --replace-needed libcrypt.so.1 libcrypt.so
 
           for dir in lib/ReSharperHost/linux-*; do
-            ln -s ${dotnet-sdk_7} $dir/dotnet
+            rm -rf $dir/dotnet
+            ln -s ${dotnet-sdk_8} $dir/dotnet
```

Comment:
As mentioned I'd keep this at sdk_7 since this is what Rider seems to be shipping with. Who knows if sdk_8 works subtly different and breaks certain functionality in the IDE.

Otherwise this looks good to me now, thanks for making all the changes!